default.nix

```diff
@@ -0,0 +1,218 @@
+{ pkgs ? import <nixpkgs> { }
+, stdenv ? pkgs.stdenv
+, lib ? pkgs.lib
+, ghostscript ? pkgs.ghostscript
+, ref ? null
+, doPdfGeneration ? true
+  # Checks whether the generated output matches the checked-in SSS32.ps.
+  # When doing development you may want to shut this off to obtain the
+  # output file that you need to check in.
+, doOutputDiff ? true
+}:
+
+let
+  src =
+    if isNull ref
+    then ./.
```

Comment:
Probably want some sort of filter source here.  I find that `./.` gets really polluted, especially with `result` symlinks from builds.

Reply:
Sure -- I suspect we actually only want the .ps and .inc files, so this should be an easy filter.

Reply:
Added a commit which filters the source.